homework/shelkovich_ilya/lookUpArrayService/src/test/java/com/tutrit/java/quickstart/service/LookUpArrayServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LookUpArrayServiceTest {
+
+    LookUpArrayService lookUpArrayService;
```

Comment:
private

Reply:
thought that in tests everything is public, thx